lib/cell-manager.coffee

```diff
@@ -0,0 +1,51 @@
+_ = require 'lodash'
+
+module.exports = CellManager =
+    removeAllBreakPoints: ->
+        editor = atom.workspace.getActiveTextEditor()
+        decorations = editor.getLineNumberDecorations({class: 'break-point'})
+        for decoration in decorations
+            decoration.marker.destroy()
+
+    removeLatestBreakPoint: ->
+        editor = atom.workspace.getActiveTextEditor()
+        decorations = editor.getLineNumberDecorations({class: 'break-point'})
+        if decorations.length isnt 0
+            _.last(decorations).marker.destroy()
+
+    addBreakPoint: ->
+        editor = atom.workspace.getActiveTextEditor()
+        row = editor.getLastCursor().getBufferRow()
+
+        marker = editor.markBufferPosition
+            row: row
+            column: 0
+            invalidate: 'never'
+
```

Comment:
Isn't the signature `::markBufferPosition(position, [options])`?

https://atom.io/docs/api/v1.8.0/TextEditor#instance-markBufferPosition